src/main/webapp/app/exercises/shared/dashboards/tutor/tutor-exercise-dashboard.component.html

```diff
@@ -226,10 +226,14 @@ <h4 *ngIf="tutorParticipationStatus === TRAINED || tutorParticipationStatus ===
                 {{ 'artemisApp.tutorExerciseDashboard.studentsSubmissions' | translate }}
             </h4>
             <div class="table-responsive" *ngIf="tutorParticipationStatus === TRAINED || tutorParticipationStatus === COMPLETED">
-                <table class="table table-striped exercise-table" *ngIf="(unassessedSubmission && unassessedSubmission?.id > 0) || submissions.length > 0; else noSubmissions">
+                <table
+                    class="table table-striped exercise-table"
+                    *ngIf="(unassessedSubmission && unassessedSubmission?.id > 0) || assessedSubmissions.length > 0; else noSubmissions"
+                >
                     <thead>
                         <tr>
                             <th>#</th>
+                            <th *ngIf="!!exercise.exerciseGroup">{{ 'artemisApp.tutorExerciseDashboard.submissionId' | translate }}</th>
```

Comment:
maybe we can do here instead of `!!exercise.exerciseGroup` add `isExamMode` like we have it in to other parts of the client.
Should define `isExamMode` as global variable and set it in `ngOnInit()`

Reply:
added global variable `isExamMode`